src/content/app/genome-browser/components/zmenu/zmenus/RegulationZmenu.tsx

```diff
@@ -38,9 +44,25 @@ type Props = {
 const RegulationZmenu = (props: Props) => {
   const { content } = props.payload;
   const genomeId = useAppSelector(getBrowserActiveGenomeId) || '';
+  const enabledCommittedSpecies = useAppSelector(getEnabledCommittedSpecies);
+  const activeSpecies = enabledCommittedSpecies.find(
+    (species) => species.genome_id === genomeId
+  );
 
   const featureMetadata = extractFeatureMetadata(props.payload);
 
+  let regulationExternalUrl;
+  if (activeSpecies) {
+    const species = activeSpecies.scientific_name
+      .toLowerCase()
+      .replace(' ', '_');
```

Comment:
Since you added a helper function, this string transformation should probably happen inside of it.